.node-version

```diff
@@ -1 +1 @@
-15.6.0
+16.13.0
```

Comment:
suggestion: please also update the instructions in `README.md`.

Reply:
16.13.0
https://github.com/ChildishGhost/einstein/blob/dev/README.md?plain=1#L15
https://github.com/ChildishGhost/einstein/blob/dev/README.md?plain=1#L21

`npm ci`
https://github.com/ChildishGhost/einstein/blob/dev/README.md?plain=1#L28